tutorials/advanced_features/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Let us now simulate the entire sequence with Amplitude noise: <s>much like what we did in the SPAM case.</s></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>